py3status/modules/sysdata.py

```diff
@@ -289,7 +334,10 @@ def sysData(self):
                 try:
                     self.py3.threshold_get_color(mem_used_percent, 'max_cpu_mem')
                 except:
-                    pass
+                    try:
+                        self.py3.threshold_get_color(swap_used_percent, 'max_cpu_mem')
+                    except:
+                        pass
```

Comment:
This is about preserving some nasty old behaviour so I think you shouldn't add to the noise.  It should really have a few comments to explain what it is doing.